sdk/identity/identity/src/msal/browserFlows/msalAuthCode.ts

```diff
@@ -43,16 +43,23 @@ export class MSALAuthCode extends MsalBrowser {
         piiLoggingEnabled: options.loggingOptions?.enableUnsafeSupportLogging,
       },
     };
+  }
 
-    // Preparing the MSAL application.
-    this.app = new msalBrowser.PublicClientApplication(
-      this.msalConfig as msalBrowser.Configuration
-    );
-    if (this.account) {
-      this.app.setActiveAccount(publicToMsal(this.account));
+  private async getApp(): Promise<msalBrowser.IPublicClientApplication> {
+    if (!this.app) {
+      // Prepare the MSAL application
+      this.app = await msalBrowser.PublicClientApplication.createPublicClientApplication(
+        this.msalConfig,
+      );
+
+      // setting the account right after the app is created.
+      if (this.account) {
```

Comment:
Not for this PR but I am going to get rid of `this.account` soon - most of the usage can be replaced with `this.getActiveAccount` 👍 😍 

nothing blocking here just an FYI

Reply:
ok got it